pkg/devfile/adapters/kubernetes/component/commandhandler.go

```diff
@@ -55,7 +55,7 @@ func (a *adapterHandler) Execute(devfileCmd devfilev1.Command) error {
 				// Creating with no spin because the command could be long-running, and we cannot determine when it will end.
 				s.Start(fmt.Sprintf("Executing the application (command: %s)", devfileCmd.Id), true)
 			case remotecmd.Stopped, remotecmd.Errored:
-				s.End(status == remotecmd.Stopped)
+				s.EndWithStatus(fmt.Sprintf("Finished executing the long running application (command: %s)", devfileCmd.Id), status == remotecmd.Stopped)
```

Comment:
Okay, after a quick chat with Philippe as well, I'm ok with the changes here. Let's hope it helps users understand what is being done here.

One comment though on the message. The command might not necessarily be a long-running one (case of short commands part of a composite run/debug command). So I would suggest changing the message to something like:

```suggestion
				s.EndWithStatus(fmt.Sprintf("Finished executing the application (command: %s)", devfileCmd.Id), status == remotecmd.Stopped)
```

Reply:
Another point discussed was, if possible, having the "Cleaning resources, please wait" message displayed last, after all the errors have been captured and handled. This would also help avoid waiting a few minutes until resources are deleted before displaying errors.
But this can be addressed in a separate issue.